extensions/control-plane/api/management-api/contract-negotiation-api/src/main/java/org/eclipse/edc/connector/api/management/contractnegotiation/transform/NegotiationInitiateRequestDtoToDataRequestTransformer.java

```diff
@@ -46,6 +47,7 @@ public Class<ContractOfferRequest> getOutputType() {
                 .consumer(URI.create("urn:connector:consumer"))
                 .provider(URI.create("urn:connector:provider"))
                 .policy(object.getOffer().getPolicy())
+                .contractEnd(ZonedDateTime.now()) // dummy end date, actual contract end will be set by the provider
```

Comment:
I think this field should be reported on the `ContractOfferDescription` and reported here, then needs to be validated by the provider in the `ContractValidationServiceImpl.validateInitialOffer`, like it has been done with the policy.

Reply:
First part done. 
For the check in `ContractValidationService`, it would be preferable to clarify first if ContractOffer should convey a duration instead of the actual end date. As it is it would force to compare a end date with a duration in the contract validation...